docker-compose.develop.yml

```diff
@@ -0,0 +1,9 @@
+---
+version: '3'
+services:
+  freqtrade_develop:
+    build:
+      context: .
+      dockerfile: "./Dockerfile"
```

Comment:
I think the develop version should use `Dockerfile.develop` ... which installs all requirements, including the ones for tests, and does not start the bot automatically (obviously then you'll probably need the compose alternatives of `-i -t` (currently not sure on the syntax for these)